features/glance.feature

```diff
@@ -0,0 +1,39 @@
+@glance
+Feature: Glance
+  As an administrator
+  I want to make sure that Glance component is working correctly
+
+  @create_kvm_image
```

Comment:
Yep, that's the only alternative I think. I will adapt the code to accept an env var for a feature `rake` task.